admin/languages.php

```diff
@@ -164,6 +164,19 @@
                                 '" . zen_db_input($coupon['coupon_description']) . "')");
         }
 
+        // create additional ez-page_description records
+        $ezpages = $db->Execute("SELECT pages_id, pages_title, pages_html_text
+                                 FROM " . TABLE_EZPAGES_CONTENT . "
+                                 WHERE languages_id = " . (int)$_SESSION['languages_id']);
+
+        foreach ($ezpages as $ezpage) {
+          $db->Execute("INSERT INTO " . TABLE_EZPAGES_CONTENT . " (pages_id, languages_id, pages_title, pages_html_text)
+                        VALUES ('" . (int)$coupon['pages_id'] . "',
+                                '" . (int)$insert_id . "',
+                                '" . zen_db_input($ezpages['pages_title']) . "',
+                                '" . zen_db_input($ezpage['pages_html_text']) . "')");

```

Comment:
Shouldn't these `$ezpages[]` vars be singular `$ezpage[]` to match the foreach?

Reply:
yes